docs/data/date-pickers/date-range-picker/FormPropsDateRangePickers.tsx

```diff
@@ -1,49 +1,41 @@
 import * as React from 'react';
-import { Dayjs } from 'dayjs';
-import TextField from '@mui/material/TextField';
-import { LocalizationProvider } from '@mui/x-date-pickers-pro';
-import { AdapterDayjs } from '@mui/x-date-pickers-pro/AdapterDayjs';
-import { DateRangePicker, DateRange } from '@mui/x-date-pickers-pro/DateRangePicker';
+import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
 import Stack from '@mui/material/Stack';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { Unstable_NextDateRangePicker as NextDateRangePicker } from '@mui/x-date-pickers-pro/NextDateRangePicker';
 
-export default function FormPropsDateRangePickers() {
-  const [value, setValue] = React.useState<DateRange<Dayjs>>([null, null]);
+function GridItem({
```

Comment:
We already have quite a few instances of these components in various docs pages.
Maybe it would make sense to extract it into its own fine? 🤔

Reply:
I started looking into a centralized implementation but did not have the time to work on it yet :+1: